exercises/practice/react/lib/react.ex

```diff
@@ -0,0 +1,41 @@
+defmodule React do
+  @doc """
+  Start a reactive system
+  """
+  @spec new(cells :: [%{atom => any}]) :: {:ok, pid}
+  def new(cells) do
+  end
+
+  @doc """
+  Return the value of an input or output cell
+  """
+  @spec get_value(cells :: pid, cell :: String.t()) :: any()
+  def get_value(cells, cell) do
+  end
+
+  @doc """
+  Set the value of an input cell
+  """
+  @spec set_value(cells :: pid, cell :: String.t(), value :: any) :: :ok
+  def set_value(cells, cell, value) do
+  end
+
+  @doc """
+  Add a callback to an output cell
+  """
+  @spec add_callback(
+          cells :: pid,
+          cell :: String.t(),
+          callback :: String.t(),
+          send :: (String.t(), any -> :ok)
+        ) :: :ok
+  def add_callback(cells, cell, callback, send) do
+  end
```

Comment:
I think we could improve argument names in the whole stub file E.g. I'm not entirely sure about the name "send". It's specific to message sending, but the callback can do anything. In practice it only makes sense if it does side-effect (because there's nobody that cares about the return value) but the side effects could be not only message sending, but also doing network requests or something 🤷 

I would propose:
`cell` -> `cell_name` (because this is the string, not the map `%{type: :input, value: 1}` and definitely not a single instance of the first argument, `cells`)
`callback` -> `callback_name` (because "callback" usually means it's a function)
`send` -> `callback` or `function` or `callback_fun`?